pkg/cli/common.go

```diff
@@ -211,6 +212,7 @@ func GetBudFlags(flags *BudResults) pflag.FlagSet {
 	fs.IntVar(&flags.Jobs, "jobs", 1, "how many stages to run in parallel")
 	fs.StringArrayVar(&flags.Label, "label", []string{}, "set metadata for an image (default [])")
 	fs.StringVar(&flags.Logfile, "logfile", "", "log to `file` instead of stdout/stderr")
+	fs.BoolVar(&flags.LogSplitByPlatform, "split-logfile-by-platform", false, "split logfile to different files for each platform")
```

Comment:
Could you shorten this to something like `--logsplit` I hate multi word options, and this might be the worst one ever.  Document the option in the help, do not need to have an explanation in the option name itself.  Also the rest of the world is not as good a typest as you.